Wikipedia/Code/NewArticleTabCoordinator.swift

```diff
@@ -135,25 +138,74 @@ final class NewArticleTabCoordinator: Coordinator {
                         relatedArticles: relatedRecords
                     )
                     becauseVM?.onTapArticle = onTapArticleAction
-
                 }
             }
-
+            
+            let dykVM = WMFNewArticleTabDidYouKnowViewModel(
```

Comment:
Now that we have the ABC group separation in place, we should only fetch did you know if the user was assigned to the group, same for because you read.
Making both calls all the time is making the new tab very slow.